b2share/modules/apiroot/views.py

```diff
@@ -40,14 +40,20 @@ def __init__(self, *args, **kwargs):
     def get(self, **kwargs):
         b2access = current_app.config.get('OAUTHCLIENT_REMOTE_APPS', {}).get(
             'b2access', {})
+        help_links = current_app.config.get('HELP_LINKS')
         data = {
             'version': __version__,
             'site_function': current_app.config.get('SITE_FUNCTION', ''),
             'training_site_link': current_app.config.get('TRAINING_SITE_LINK', ''),
             'b2access_registration_link': b2access.get('registration_url'),
             'b2note_url': current_app.config.get('B2NOTE_URL'),
             'terms_of_use_link': current_app.config.get('TERMS_OF_USE_LINK'),
-            'help_links': current_app.config.get('HELP_LINKS')
+            'help_links': {
+                'issues': help_links.get('issues', ''),
+                'user-guide': help_links.get('user-guide', ''),
+                'rest-api': help_links.get('rest-api', ''),
+                'search': help_links.get('search', '')
+            }
```

Comment:
help_links are also defined in config.py.
Maybe remove these if the ones in config.py are enough?